test/integration/targets/listen_ports_facts/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+# Test playbook for the listen_ports_facts module
+# (c) 2019, Nathan Davison <ndavison85@gmail.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- name: install netstat and netcat on deb
+  apt:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - net-tools
+    - netcat
+  when: ansible_os_family == 'Debian'
+
+- name: install netstat and netcat on rh < 7
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - net-tools
+    - nc.x86_64
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version|int < 7
+
+- name: install netstat and netcat on rh >= 7
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - net-tools
+    - nmap-ncat
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version|int >= 7
+
+- name: start UDP server on port 5555
+  shell: nc -u -l 5555 &
```

Comment:
It's probably wiser to make these an async tasks.
You can clean up  async tasks properly as they are managed by Ansible.